example_app/web/templates/page/index.html.eex

```diff
@@ -37,3 +54,9 @@
     </ul>
   </div>
 </div>
+
+<script type='text/javascript'>
+  $('.logout-link').click(function() {
+    $.ajax({url:"/logout", method: "DELETE"}).done(function() { window.location.reload(); });
+  });
```

Comment:
I'm curious - is there a benefit to using this approach over something like `<%= link "Logout", to: logout_path(@conn, :delete), method: :delete %> ? The existing approach certainly has more straight forward markup.